btc-deposit/README.md

```diff
@@ -1,31 +1,16 @@
-
 ## Bitcoin deposit service
-Bitcoin Deposit service(or simply `btc-deposit`) is here to listen to Bitcoin blockchain transactions and increase clients balances in Iroha blockchain. 
+The Bitcoin deposit service(or simply `btc-deposit`) is here to listen to Bitcoin blockchain transactions and increase clients balances in Iroha blockchain. 
 
 ### Simplified flow
-1) The service creates Bitcoin blockchain listeners that listen to transactions where coins were sent to our clients. The listener will wait until the transaction of interest hits at least 6 blocks. Once it happens, an Iroha transaction is created. 
-2) Then it starts Bitcoin blockchain downloading process (only headers are stored on the disk)
-3) After that, the service increases client's balance in Iroha blockchain if needed. This 'increase' works in a multisignature fashion. So every node must create the same Iroha transaction. Bitcoin block time is used as a source of time for the Iroha 'increase balance' transaction. Doing that we can guarantee that every node will use the same time.
+1) The service creates Bitcoin blockchain listeners that listen to transactions where coins were sent to our clients. The listener waits until the transaction of interest reaches at least 6(or another amount of blocks which may be specified) blocks in depth. Once it happens, an Iroha 'increase balance' transaction is created. This 'increase' works in a multisignature fashion. So every node must create the same Iroha transaction. Bitcoin block time is used as a source of time for the Iroha 'increase balance' transaction. 
```

Comment:
waits for the transaction of interest to reach, least 6(or another – no space, an Iroha  - definitely no an, an Iroha